lib/project_types/script/loaders/project.rb

```diff
@@ -32,7 +32,7 @@ def self.handle_error(error, context:, env_file_present:)
         else
           properties_hash = { api_key: "--api-key", secret: "--api-secret" }
           missing_options = error.properties.map { |p| properties_hash[p.name] }.compact.join(", ")
-          raise ShopifyCLI::Abort, context.message("script.error.missing_push_options", missing_options)
+          raise ShopifyCLI::Abort, context.message("script.error.missing_push_options", missing_options, ShopifyCli::TOOL_NAME).
```

Comment:
I _think_ this is causing the failing CI
```suggestion
          raise ShopifyCLI::Abort, context.message("script.error.missing_push_options", missing_options, ShopifyCli::TOOL_NAME)
```